.github/workflows/build.yml

```diff
@@ -135,3 +137,13 @@ jobs:
           redhat_tag: quay.io/redhat-isv-containers/68961b49ae4b51ff35f52b77:${{env.version}}-ubi
           tags: |
             docker.io/hashicorp/${{env.repo}}:${{env.version}}-ubi
+
+      - name: Docker Build (Action) ${{ matrix.arch }}
+        # linux/s390x images can only ever be published the Red Hat catalog. In the future they may be pushed to ICR as well.
+        if: "${{ matrix.arch == 's390x'}}"
```

Comment:
Same idea here:

```suggestion
        if: ${{ matrix.arch == 's390x'}}
```

Reply:
updated in 3d1660a2eac2078ae9c4fe60fb9bcff83bacd775